### PR TITLE
Add remaining repositories to automation

### PR DESCRIPTION
**Pull request by samuelallan72**
- minimal github repo settings
- codeowners file

### CODE REVIEW

File: terraform-plans/configs/charmed-openstack-exporter-snap_main.tfvars
Comment (chanchiwai-ray) [inline terraform-plans/configs/charmed-openstack-exporter-snap_main.tfvars:1]: Can we not change this file if we are only adding remaining repository to automation repo?

**Review by chanchiwai-ray — CHANGES_REQUESTED** [review]
Thanks, I have one question

File: terraform-plans/configs/charmed-openstack-exporter-snap_main.tfvars
Comment (samuelallan72) [reply terraform-plans/configs/charmed-openstack-exporter-snap_main.tfvars:1]: Good catch, I don't know how that got changed :thinking:

Comment (samuelallan72) [reply terraform-plans/configs/charmed-openstack-exporter-snap_main.tfvars:1]: Updated :)

**Review by chanchiwai-ray — APPROVED** [review]
LGTM, however, I do notice `homepage_url` in the github repo is overrided to `null` (e.g. openstack-exporter-operator). This is not a big problem since we don't have many project specify a `homepage_url`, but flagged a potential issue to me: it's hard to review what's being changes to the repo setting without looking at the workflow result.

File: .github/workflows/terraform-apply.yaml [hunk 1]
```diff
@@ -24,17 +24,50 @@ jobs:
         matrix:
           repository:
             # The naming convention of the files is $REPO_$BRANCH.tfvars
+            - bootstack-actions_main
```
Comment (Pjack) [inline .github/workflows/terraform-apply.yaml:27]: Bootstack-actions will be deprecated. But it's not a big issue to apply to this project too.  I am fine with that.

**Review by Pjack — APPROVED** [review]

File: .github/workflows/terraform-apply.yaml (hunk 1 above)
Comment (samuelallan72) [reply .github/workflows/terraform-apply.yaml:27]: Thanks :)

**Pull request merged**